crates/cli/src/subcommands/generate/mod.rs

```diff
@@ -89,11 +89,12 @@ pub fn cli() -> clap::Command {
                 .value_parser(clap::value_parser!(Language))
                 .help("The language to generate"),
         )
-        .arg(Arg::new("delete_files").long("delete-files").action(SetTrue).help(
-            "Delete outdated generated files whose definitions have been \
-             removed from the module. Prompts before deleting unless --force is \
-             supplied.",
-        ))
+        .arg(
+            Arg::new("delete_files")
+                .long("clear-target-directory")
```

Comment:
The rest of the changes here are non-controvertial other than this one - I might recommend pulling this out so that we can get the rest of these changes in through this PR. We can discuss more what we want to do about `--delete-files`.